docs/conf/atlantic/2024/news/announcing-speakers.rst

```diff
@@ -0,0 +1,56 @@
+:template: {{year}}/generic.html
+
+.. post:: June 06, 2024
+   :tags: atlantic-2024, speakers, tickets, shirts
+
+Announcing speakers and talks
+=============================
+
+Greetings, documentarians! Today we're excited to announce that we've finalized our speakers for the next virtual {{city}} conference!
+
+Thank you so much to everyone who took the time to send us a proposal – we literally wouldn't have an event without you!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. 
+Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
```

Comment:
Is there a topic that the description "odd whimsical" especially applies to this time? I did not think so at quick glance.